src/Command/PackageCommand.php

```diff
@@ -131,16 +132,22 @@ public function execute(InputInterface $input, OutputInterface $output)
 
         // Sign the README.md file
         $signingKey = $this->gitHelper->getSigningKey();
-        (new Process(
+        $this->runProcess(
             'gpg --yes --digest-algo SHA256 -u ' . $signingKey . ' --clearsign --output README.md README_unsigned.md',
             $artefactsDirectory
-        ))->run();
+        );
         unlink($artefactsDirectory . '/README_unsigned.md');
 
         // Legacy code
         // Show MD5 hashes and SHA1 hashes
-        (new Process('md5 -r *.gz *.zip', $artefactsDirectory))->run();
-        (new Process('shasum  *.gz *.zip', $artefactsDirectory))->run();
+        $this->io->note(
+            "MD5 hashes of the artefacts:\n"
+            . $this->runProcess('md5sum *.gz *.zip', $artefactsDirectory)->getOutput()
```

Comment:
Those binaries don't exist on macOS, unless installed via brew. We probably need some kind of a "fallback chain" here...

Reply:
Spotted that as well... actually we'd have `CHECKSUM_MD5_COMMAND` in `.env` for that...

Reply:
Ah, it was hard-coded to `md5` before. That's why it worked...